upup/models/cloudup/resources/addons/storage-openstack.addons.k8s.io/k8s-1.16.yaml.template

```diff
@@ -333,7 +333,7 @@ spec:
             - name: CLOUD_CONFIG
               value: /etc/kubernetes/cloud.config
             - name: CLUSTER_NAME
-              value: kubernetes
+              value: {{ ClusterName }}
```

Comment:
hmm, what will happen to existing installations? It might work correctly to newer one, but what about old ones?

Reply:
tried to answer this in one place, here: https://github.com/kubernetes/kops/pull/15095#issuecomment-1415685298